recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.0" %}
+{% set sha256 = "06ec0990d362963fc5e02ad64bc90d99ad0d226014354de4566af267d33443b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
+    - cmake
+    - make
+    - go
+    - maven
+  host:
+    - {{ compiler('cxx') }}
```

Comment:
Do you really need it here? If so, add a comment as this is unusual.

Reply:
It appears to be needed as nvcc activation script needs it to determine conda env sysroot. attn: @jakirkham

Reply:
You shouldn't need it if you change https://github.com/conda-forge/nvcc-feedstock/blob/master/recipe/install_nvcc.sh#L71 from $CC to $GCC.

Reply:
@isuruf Are you suggesting that $GCC is always available in `host` section environment? Even when no `compiler` dependency is used. If so, I'll create a `nvcc-feedstock` issue.

Reply:
You could do, `${GCC:-${CC}}`

Reply:
I still don't see how that would help if no `compiler` dependency is defined within `host`: `GCC` or `CC` has to be defined by some package.
IMHO, `nvcc-feedstock` should not use `CC` to determine the conda env sysroot as it can be determine from conda env variables alone.

Reply:
True. We can just change that to `$PREFIX/$HOST/sysroot` at build time. Can you send a PR?

Reply:
Yes: https://github.com/conda-forge/nvcc-feedstock/pull/31